app/models/post.rb

```diff
@@ -1,2 +1,5 @@
 class Post < ApplicationRecord
+
+  belongs_to :users, class_name: 'User', :foreign_key => "user_id"
```

Comment:
can you add comment to say what this is for reference?

Reply:
done